packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -454,31 +455,34 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 if (this.hostPolicy.summarizerClient && trees && blobs) {
                     const blobsIdToPathMap: Map<string, string> = new Map();
                     let appCommit: string | undefined;
-                    let appTree: string | undefined;
+                    let appTreeId: string | undefined;
 
                     for (const [key, treeVal] of this.treesCache.entries()) {
-                        if (!appCommit && !appTree) {
+                        if (!appCommit && !appTreeId) {
                             for (const entry of treeVal.entries) {
                                 if (entry.path === ".app") {
                                     if (entry.type === "commit") {
                                         // This is the unacked handle of the latest summary generated.
                                         appCommit = entry.id;
                                     }
                                     if (entry.type === "tree") {
-                                        appTree = entry.id;
+                                        appTreeId = entry.id;
                                     }
                                     break;
                                 }
                             }
-                            assert(!!appCommit || !!appTree); // .app commit or tree should be first entry in first entry.
+                            assert(!!appCommit || !!appTreeId); // .app commit or tree should be first entry in first entry.
                         }
                         for (const entry of treeVal.entries) {
                             if (entry.type === "blob") {
                                 blobsIdToPathMap.set(entry.id, key === appCommit ? `/.app/${entry.path}` : `/${entry.path}`);
                             }
                         }
                     }
-
+                    assert(id !== undefined, "App tree should have a id!");
```

Comment:
any chance you can rename id to something more descriptive?
Comment is a bit misleading as there is already appTreeId, so I assume it's not "app tree id" as assert suggests.
Or if it is, something (naming, comment) should clearly articulate different between this id and appTreeId

Reply:
Removed this piece of code as we now build the cache during getSnapshotTree call.